packages/fxa-auth-server/lib/senders/emails/styles.ts

```diff
@@ -0,0 +1,61 @@
+export const style = `
+.header-text div{
+    font-family: sans-serif !important;
+    font-size: 21px !important;
+    line-height: 29px !important;
```

Comment:
I'm guessing all of the ones marked `!important` have to be done that way?

Ooh, let's definitely use this as an opportunity to make the styles more consistent. If you look at the design guide in Settings (`yarn storybook` in `fxa-settings`), you'll see we're going by units of `4px`. It's unfortunate [some email clients don't seem to support rem](https://www.caniemail.com/features/css-unit-rem/) which is what Tailwind/the design guide uses, so we'll have to use `px` or `em`. Let's try to set some patterns here by checking out the Settings design guide - I kind of feel like we could use all `em`s here to better translate from the design guide rather than using `px`, curious what @jodyheavener thinks.

Reply:
I'm torn about this. I do agree that it would be nice to make these styles consistent, but I am resistant for two reasons:

1. They were written by someone, and given the specificity it seems like they were intentional
2. We aren't updating designs in this project, and while these are small changes they feel like a slippery slope, will also probably require a more thorough design review

Open to pushback though, I think if we're gonna do this we should make a formal plan for it!

Reply:
Hmm, but if we had chosen the React and Tailwind option, it would have forced us to style them for consistency. To me, one big win from "redoing" these was consistent styles - I feel like it would be a lost opportunity if we keep the random values.

> given the specificity it seems like they were intentional

As someone that has updated our emails and styles before... not really, lol. Most of the time, the devs just pick a px value they think looks good and they just go with it. I feel like it would be fine to round to the nearest unit of 4 for padding and margin, and then we can come up with font-sizes and corresponding line-heights very similar to what we're getting out of Tailwind.